app/javascript/components/visualization/controls/AnnotationSelector.jsx

```diff
@@ -53,7 +56,12 @@ export default function AnnotationControl({
 
   return (
     <div className="form-group">
-      <label className="labeled-select">Annotation
+      <label className="labeled-select">Annotation&nbsp;
+        <a className="action help-icon"
+           data-toggle="tooltip"
+           data-original-title="Use this menu to specify how the data are grouped or colored">
```

Comment:
Briefer, for consideration:

```suggestion
           data-original-title="Select how data are grouped or colored">
```